### ISSUE TITLE
Ethereum Discovery Error

### ISSUE THREAD

**Issue by Kevin-h-j** [trezor/trezor-suite#3191]
I am trying to access my ETH wallet but I get a message saying there is an account discovery error. What did this mean? Where is my wallet?

**Comment by prusnak** [comment]
> Trezor Suite

Transfering to the trezor-suite repository where bugs about Suite should be reported: https://github.com/trezor/trezor-suite

**Comment by mroz22** [comment]
Dear @Kevin-h-j 
we appreciate your report but we would really need some more detailed explanation.
1. Could you provide detailed steps that we could follow to simulate this issue? 
1. Did yo try to load another ethereum account? Does it happen everywhere?

**Comment by prusnak** [comment]
@Kevin-h-j don't you have Tor enabled by any chance?

**Comment by Kevin-h-j** [comment]
@mroz22 The issue appears to have resolved itself but to answer your questions

(1) I connected my wallet to my computer via USB. I entered my pin and opened the Trezor Suite app on my desktop. I could see my Bitcoin wallet but the Ethereum wallet indicted there was a discovery error. I clicked retry and was still unable to find it. This morning it seems to be working

(2) I only had the one Ethereum account. I was able to see my Bitcoin wallet but not my Ethereum wallet.

@prusnak I do have Tor enabled.

Thanks for the help
